pallets/schemas/src/tests.rs

```diff
@@ -160,3 +170,23 @@ fn get_non_existing_schema_by_id_should_return_none() {
 		assert_eq!(res.as_ref().is_none(), true);
 	})
 }
+
+#[test]
+fn validate_schema_is_acceptable() {
+	new_test_ext().execute_with(|| {
+		let test_str_raw = r#"{"name":"John Doe"}"#;
+		let result = SchemasPallet::ensure_valid_schema(&Vec::from(test_str_raw.as_bytes()));
+		assert_ok!(result);
+	});
+}
+
+#[test]
+fn reject_null_json_schema() {
+	new_test_ext().execute_with(|| {
+		assert_noop!(SchemasPallet::ensure_valid_schema(
+			create_bounded_schema_vec("null")
+		), Error::<Test>::InvalidSchema);
+	})
+}
+
+
```

Comment:
This needs a malformed JSON negative test, even a simple one although it’s kind of testing the serde parser